rebar.config.sample

```diff
@@ -13,12 +20,6 @@
 %% Erlang compiler options
 {erl_opts, [no_debug_info,
             {i, "myinclude"},
-            %% directories containing source files
-            {src_dirs, ["src", "src2", "src3"]},
```

Comment:
Yeah. It's still supported. I just felt it would be a better idea to slowly direct people to not using it? I can put it back in.